.github/workflows/ci-js.yml

```diff
@@ -0,0 +1,78 @@
+name: CI JS
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    types: [opened, edited, synchronize]
+
+jobs:
+  build:
+    name: build and test
+    timeout-minutes: 15
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [
+          ubuntu-latest,
+          macos-latest,
+          # windows-latest,
```

Comment:
I tried to track down previous runs, got a link? (Don't mind tabling, but I _do_ want to see it.)

Reply:
https://github.com/vercel/turborepo/runs/7777949925?check_suite_focus=true

I tthink the issue is that `yarn -v` is erroring or the `npm i -g yarn` isn't working. Something for a followup PR